.scaffold/tests/phpunit/fixtures/init/shell/AGENTS.md

```
@@ -66,9 +66,10 @@
 -```
 -
 -
--```bash
- # Shell script tests
- ./tests/bats/node_modules/bats/bin/bats tests/bats/
+ ## NodeJS Application Architecture
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)